### DISCUSSION TITLE
getting Killed response

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#2943] [Help]
NewMagickWand()
MagickReadImage()
MagickResizeImage()
MagickWriteImages()
DestroyMagickWand()

Im calling these method  in parallelly 
Im getting killed as response 
Im not getting any other message
What will be reason ?

**Comment by snibgo** [comment]
I suggest you show a complete but minimal program that shows the problem.

**Comment by urban-warrior** [comment]
Perhaps its the dreaded OOM killer.  That may indicate your threaded application is consuming too much memory.  You can fix this by perhaps reducing the number of threads or forcing the pixel cache to disk (to reduce memory consumption) with MagickSetResourceLimit().   See https://imagemagick.org/script/architecture.php#tera-pixel.
